PR for BXMSDOC-7100: Update building custom dashboard widgets doc based on BAPL-1785 (#102)
kaldesai/kie-docs: doc-content/enterprise-only/pages/building-custom-dashboard-widgets-running-heatmap-components-proc.adoc

> [id='building-custom-dashboard-widgets-running-heatmap-components-proc_{context}']

Reviewer: ```suggestion
[id='proc-building-custom-dashboard-widgets-running-heatmap-components_{context}']
```

> = Running heatmap components

Reviewer 2: Hi @barbora ,
Can you please review this section

Author: Hey @sthundat can you please delete or edit this comment? I guess you have tagged the wrong Barbora here. The correct one is @barboras7. 

I am unable to remove @barbora from the thread.

> = Running heatmap components

Reviewer: ```suggestion
= Executing a heatmap component
```

Author: Done

You can run the internal components locally, outside of {CENTRAL} for testing purpose only. The same API is used to create external components. You can build your own components using an external components.
For more information about external components, see <<building-custom-dashboard-widgets-external-components-con_{context}>>

To run the specific heatmap component, complete the following steps:

.Prerequisites

* npm is installed on your system. For more information about installing npm, see https://docs.npmjs.com/downloading-and-installing-node-js-and-npm[Downloading and installing Node.js and npm].

> * Yarn is installed on your system. for more information about installing yarn, See https://classic.yarnpkg.com/en/docs/install/#debian-stable[2 - Installation].

Reviewer: ```suggestion
* You have installed Yarn on your system. For more information about installing Yarn, See https://classic.yarnpkg.com/en/docs/install/#debian-stable[Yarn Installation].
```

.Procedure

> . Go to `appformer/dashbuilder/dashbuilder-shared/dashbuilder-js` directory.

Reviewer 3: This is not providing enough information. I would add into the "Prerequisites" section that to run the components outside of business central, the user needs to clone the [Appformer repository](https://github.com/kiegroup/appformer/) and after that, they can navigate in the command line into the directory that you are mentioning in this step

Author: Added this in the **Prerequisites** section.

> . In `dashbuilder-js` directory, open the terminal and enter the following command:

Reviewer: ```suggestion
. Open the terminal in the `dashbuilder-js` directory and enter the following command:
```

+
[source]
----
yarn run init && yarn run build:fast
----
+

> In the `dashbuilder-js/packages` directory, following components are present:

Reviewer: ```suggestion
You can see the following components are present In the `dashbuilder-js/packages` directory:
```

* processes-heatmaps-component
* process-heatmap-component
* logo-component
* heatmap-component

. Go to `dashbuilder-js/packages` directory, open the desired heatmap component and enter the following command in the terminal:
+
[source]
----
yarn run start
----
. To access the heatmap component, enter http://localhost:9001/ in a web browser.
+

> You can see the selected heatmap component on a web browser.

Reviewer 3: This is not exactly correct. There is also a logo component (and that is not a heatmap component), hence I would suggest to skip the word heatmap here (and also in some sentences above)

Author: Done.
